apps/web/src/app/app/[workspaceSlug]/(dashboard)/notifications/(overview)/_components/channel-table.tsx

```diff
@@ -11,8 +10,8 @@ interface ChannelTable {
 }
 
 export default function ChannelTable({ workspace, disabled }: ChannelTable) {
-  const isPagerDutyAllowed = getLimit(workspace.plan, "pagerduty");
```

Comment:
We should keep the `getLimit` extraction.

That way, if the `workspace.limits = "{}"` we use the default config that we have in `@openstatus/plan` - makes it more dynamic and we only update the database workspaces of paid users.

It gives us a layer of flexibility that we will loose otherwise.